<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSettings</class>
 <widget class="QDialog" name="ImageSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>230</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>320</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Image</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="imageResolutionBox"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Image Format:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QComboBox" name="imageCodecBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Quality:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="imageQualitySlider">
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
